lib/rdoc/parser/rbs.rb

```diff
@@ -0,0 +1,175 @@
+# frozen_string_literal: true
+
+require 'rbs'
+
+module RDoc
+  class Parser
+    class RBS < Parser
```

Comment:
Move the definition of `RDoc::Parser::RBS` class to `lib/rdoc/discover.rb` to encapsulate everything related to Doc plugin mechanism there.